apps/extension/src/routes/popup/approval/approve-deny.tsx

```diff
@@ -22,6 +24,11 @@ export const ApproveDeny = ({
       <Button size='lg' variant='destructive' onClick={deny}>
         Deny
       </Button>
+      {ignore && (
+        <Button size='lg' variant='destructive' className='bg-black' onClick={ignore}>
+          Ignore Site
+        </Button>
+      )}
```

Comment:
This action does not feel very standard across crypto wallets. Any chance you know how other wallets are handling this? I kinda feel an exponential cooldown backoff achieves the same result.

Reply:
if this needs to be changed i think it can be a separate issue, would like to prioritize merge for the ability to manage settings

Reply:
notably, exponential backoff would require more state.

it is very standard for the browser to use this 'approve/deny/ignore' trio for webcam/audio permission attitudes, for example. i am imitating that example.

Reply:
okay, now using state to conditionally display an ignore button, if a previous deny exists